subsys/fb/Kconfig

```diff
@@ -13,12 +13,26 @@ menuconfig CHARACTER_FRAMEBUFFER
 
 if CHARACTER_FRAMEBUFFER
 
+
+choice
+	prompt "Fonts in use"
+	default CHARACTER_FRAMEBUFFER_USE_DEFAULT_FONTS if SSD1673 || SSD1306
```

Comment:
Is it  not an option to always select the default font and where need select any other supported font through Kconfig.

Reply:
@vanwinkeljan , 
![image](https://user-images.githubusercontent.com/800198/59992788-1394a600-9680-11e9-97f5-48cfe79af03b.png), this option is prompt a default font option and use can choose other existing fonts manually, currently there only two fonts supported. Can you give me more hints on how to change?

Reply:
Just keep a single default (```default CHARACTER_FRAMEBUFFER_USE_DEFAULT_FONTS```) and dont check for display type

Reply:
@vanwinkeljan even the fonts are not applicable to default display panel?

Reply:
It just does not feel right that for every new display we add we need to update the CFB Kconfig.
I would assume the default font is the default font for all displays and if another font is need it needs to be put in an overlay

Reply:
The situation is that initial font is a tiled one and the algorithm to display it is quite different from a plain text normal font table generated. I think the difference is only for the panel type, so there will not be many fonts added. as far as I know only tiled font and plain font.

Reply:
Problem is not that there will be many fonts but a number of growing display controller drivers.
Dont know how to solve this in Kconfig (@SebastianBoe / @galak any suggestions)

Also it is now very confusing you have a Kconfig called `CHARACTER_FRAMEBUFFER_USE_DEFAULT_FONTS` which is catulay selects a specific font.
Would it not be better to provide 3 options in the choice:
 * default tiled font
 * default font
 * custom font

Reply:
ok, let me change to default
* default tiled font
* default font
* custom font